src/main/resources/languages/english.xml

```diff
@@ -1872,7 +1872,7 @@
 
 	<string>
 		<key>OpenPaperSettings</key>
```

Comment:
This is fine.  How do we choose good key names?  I wonder if we should group them by the class that calls them, like so:

    <key>Makelangelo.OpenPaperSettings</key>

Reply:
It is a good idea to group them by window, a lot of work...

Reply:
window and class should be a 1:1 relationship.  
the window name might change with language selected.  
the class name should rarely change.

bonus: then we can run a test that loads the xml file, checks that all the keys exist in the class file, and throws a warning if an unused language key/string is found.

Reply:
yes, you are correct